data/strings/strings.txt

```diff
@@ -5429,6 +5429,88 @@
     zh-Hans = 语音指导
     zh-Hant = 語音指示
 
+  [pref_tts_street_names_title]
+    comment = Settings «Route» category: «Tts announce street names» title
+    tags = android,ios
+    en = Announce Street Names
+    ar = أعلن أسماء الشوارع
+    be = Абвяшчаць назвы вуліц
+    bg = Обявете имена на улици
+    ca = Anuncia els noms dels carrers
+    cs = Oznamujte názvy ulic
+    da = Annoncer gadenavne
+    de = Straßennamen ankündigen
+    el = Ανακοινώστε ονόματα οδών
+    es = Anunciar nombres de calles
+    et = Teatage tänavanimed
+    eu = Kaleen izenak iragarri
+    fa = نام خیابان ها را اعلام کنید
+    fi = Ilmoita kadunnimet
+    fr = Annoncer les noms de rue
+    he = הכריזו על שמות רחובות
+    hu = Az utcanevek felolvasása
+    id = Umumkan Nama Jalan
+    it = Annunciare i nomi delle strade
+    ja = 通りの名前を発表
+    ko = "거리 이름 발표"
+    nb = Kunngjør gatenavn
+    nl = Straatnamen bekendmaken
+    pl = Wymawiaj nazwy ulic
+    pt = Anunciar nomes de ruas
+    pt-BR = Anunciar nomes de ruas
+    ro = Anunțați numele străzilor
+    ru = Проговаривать названия улиц
+    sk = Oznámte názvy ulíc
+    sv = Tillkännage gatunamn
+    sw = Tangaza Majina ya Mitaa
+    th = ประกาศชื่อถนน
+    tr = Sokak İsimlerini Duyur
+    uk = Проговорювати назви вулиць
+    vi = Thông báo tên đường phố
+    zh-Hans = 宣布街道名称
+    zh-Hant = 宣布街道名稱
+
+  [pref_tts_street_names_description]
+    comment = Settings «Route» category: «Tts announce street names» description
+    tags = android,ios
+    en = When enabled, the name of the street or exit to turn onto will be spoken aloud.
+    ar = عند التمكين ، سيتم نطق اسم الشارع أو المخرج الذي سيتم تشغيله بصوت عالٍ.
```

Comment:
@rtsisyk has added `az` translation. It would be great to add it too. Can you please also check if some other translations were added? Does it make sense to auto-generate them (e.g. hi) so users will fix it later?

Reply:
Yes all my added translations that haven't been human reviewed are automatically translated. I've added `az = Küçə adlarını elan edin` and `Aktiv edildikdə, dönmək üçün küçənin və ya çıxışın adı ucadan səsləndiriləcək.` hopefully that's acceptable for @rtsisyk ?

When I started this PR some languages didn't have much support so that's the only reason any language would be missing. I'll add that too.

Reply:
@1ec5 I replied elsewhere but it got attached to an unrelated comment... there are facilities in Github to see which language review team has checked a PR but the quickest way is to see which languages have been debated here in the comments. You're correct that a more formal language review process/team would be ideal, I haven't been around long enough to know what the status is.

Reply:
Any translation management service would have that functionality, including Weblate, so the answer could just be that a TMS will be adopted sooner or later.